src/plugins/data/public/ui/saved_query_flyouts/open_saved_query_flyout.tsx

```diff
@@ -33,7 +33,7 @@ export interface OpenSavedQueryFlyoutProps {
   onClose: () => void;
   onQueryOpen: (query: SavedQuery) => void;
   handleQueryDelete: (query: SavedQuery) => Promise<void>;
-  queryStringManager: QueryStringManager;
+  queryStringManager: QueryStringContract;
```

Comment:
we actually shouldn't need this at all. since it's apart of the the data public folder the query string manager can be accessed from the fetchAllSavedQueriesForSelectedTab callback like

```typescript
import { getQueryService } from '../../services';

const fetchAllSavedQueriesForSelectedTab = useCallback(async () => {
    setIsLoading(true);
    try {
      const queryStringManager = getQueryService().queryString;
      const query = queryStringManager.getQuery();
```

Reply:
Oh thanks for that context! Revised.